edx_proctoring/static/proctoring/templates/course_allowances.underscore

```diff
@@ -0,0 +1,58 @@
+<span class="tip"> <%- gettext("Add Allowance for User: ") %>
+    <span>
+        <a id="add-allowance" href="#"  class="add blue-button">+ <%- gettext("Add Allowance") %></a>
+    </span>
+</span>
+<% var is_allowances = proctored_exam_allowances.length !== 0 %>
+<% if (is_allowances) { %>
+
+<div class="wrapper-content wrapper">
+   <section class="content">
+
+      <table class="allowance-table">
+         <thead>
+            <tr class="allowance-headings">
+               <th class="exam-name"><%- gettext("Exam Name") %></th>
+               <th class="username"><%- gettext("Username") %></th>
+               <th class="email"><%- gettext("Email") %></th>
+               <th class="allowance-name"><%- gettext("Allowance Name") %> </th>
```

Comment:
@catong changed this to "Allowance Type"  
https://github.com/edx/edx-proctoring/pull/74